### PR TITLE
Rename unit tests under k8s.io/apiserver

### PR DESCRIPTION
**Pull request by oomichi**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

These unit test names were different from common naming way which
looks like "<module name>_test.go". This renames them for the way.

**Special notes for your reviewer**:

Now we are checking unit test coverage for each module, and
this issue was found during the investigation.

**Does this PR introduce a user-facing change?**:
```NONE```

### CODE REVIEW

File: staging/src/k8s.io/apiserver/pkg/apis/example2/install/BUILD [hunk 1]
```diff
@@ -21,7 +21,7 @@ go_library(
 
 go_test(
     name = "go_default_test",
-    srcs = ["roundtrip_test.go"],
+    srcs = ["install_test.go"],
```
Comment (mourya007) [inline staging/src/k8s.io/apiserver/pkg/apis/example2/install/BUILD]: Looks like  we can update the same here https://github.com/kubernetes/kubernetes/blob/2cbb16bc8dd456c5db72c1667926abdbc87c32c7/staging/src/k8s.io/apiserver/pkg/apis/example/install/BUILD#L24

Comment (oomichi) [reply staging/src/k8s.io/apiserver/pkg/apis/example2/install/BUILD]: Oh, I missed that.
Thanks for your review!

**Comment by mourya007** [comment]
cc @liggitt

**Comment by liggitt** [comment]
thanks for the PR, but there are a few reasons I think the current names should stay:
* it preserves git history
* in some cases (like the roundtrip and whitebox names), it conveys meaning about what types of tests should be included in that file

/close

**Pull request closed without merging**